### DISCUSSION TITLE
How i can capture code suggestion from Copilot in VSCode ??

### DISCUSSION THREAD

**Discussion by FabianMellaRios** [community/community#38226] [Copilot Conversations]
I want to develop an application for educational purposes powered by Github copilot, is there a way to capture the suggestions?

**Comment by ossanna16** [comment]
Hi there @FabianMellaRios and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)

**Comment by iamcjwilliams** [comment]
@FabianMellaRios Thank you for the suggestion, I will inform our copilot team!
